src/peft/tuners/boft/layer.py

```diff
@@ -316,10 +316,7 @@ def update_layer(
             boft_block_num = int(self.in_features // boft_block_size)
 
         else:
-            raise ValueError(
-                f"You can only specify either boft_block_size ({boft_block_size}) or boft_block_num ({boft_block_num}), but not both simultaneously or setting both"
-                "to be 0, because boft_block_size x boft_block_num != in_features."
-            )
+            raise ValueError("Unknown error!")
```

Comment:
So this code should not be reachable. Still, let's use a more helpful error message. My proposal:

"Something went wrong, please report this error: https://github.com/huggingface/peft/issues".